src/dmd/root/filename.d

```diff
@@ -1192,8 +1192,8 @@ version(Windows)
         }
         // actually do the conversion
         const length2 = MultiByteToWideChar(
-            0 /*codepage*/, 0 /*flags*/, &str[0], cast(int)str.length, &ret[0], cast(int)length);
-        assert(str.length == length2); // should always be true according to the API
+            codepage, 0 /*flags*/, &str[0], cast(int)str.length, &ret[0], length);
+        assert(length == length2); // should always be true according to the API
```

Comment:
The previous assertion was that the number of narrow input chars matches the number of wide output wchars.